.yamato/upm-ci-webrtc-packages.yml

```diff
@@ -143,27 +205,50 @@ test_{{ package.name }}_{{ param.platform }}_{{ param.backend }}_{{ platform.nam
       mkdir -p upm-ci~/test-results/
       scp -i ~/.ssh/id_rsa_macmini -o "StrictHostKeyChecking=no" -r bokken@$BOKKEN_DEVICE_IP:/Users/bokken/com.unity.webrtc/WebRTC~/test-results/ upm-ci~/test-results/
       exit $UTR_RESULT
-  triggers:
-    branches:
-      only:
-      - "/.*/"
-      except:
-      - "master"
   artifacts:
-    {{ package.name }}_il2cpp_{{ editor.version }}_{{ platform.name }}_test_results: 
+    {{ package.name }}_il2cpp_{{ editor.version }}_{{ target.name }}_test_results: 
       paths:
         - "upm-ci~/test-results/**/*"
   dependencies:
     - .yamato/upm-ci-{{ package.name }}-packages.yml#pack_{{ package.name }}
 {% endfor %}
-
-
 {% endif -%}
 {% endfor %}
 
+test_{{ package.name }}_nogpu:
+  name : Test {{ package.packagename }} all platforms without GPU
+  dependencies:
+    {% for target in test_targets %}
+    {% for editor in editors %}
+    {% for param in target.test_params %}
+    {% if target.is_gpu == "false" -%}
+    - .yamato/upm-ci-{{ package.name }}-packages.yml#test_{{ package.name }}_{{ param.platform }}_{{ param.backend }}_{{ target.name }}_{{ editor.version }}
+    {% endif -%}
+    {% endfor %}
+    {% endfor %}
+    {% endfor %}
+
+test_{{ package.name }}_gpu:
+  name : Test {{ package.packagename }} all platforms with GPU
+  triggers:
+    branches:
+      only:
+      - "/.*/"
+      except:
+      - "master"
```

Comment:
I think this change for avoiding use `gpu vm`.
so, `tiggers` is defined on `test_{{ package.name}}_nogpu`, isn't it?

Reply:
In the end, we should do it, but it has not done yet.
Currently, the test would be failed without GPU.